scripts/sample_data/loaddata.py

```diff
@@ -0,0 +1,74 @@
+# load.py -- load sample data into an Ashlar Blueprint instance
```

Comment:
Does this get run as part of the normal setup process? I think I noticed data that I didn't remember entering myself. It'd be good to add something about this to the README.

Reply:
It does! It gets run as part of `scripts/update`. I'll add a short note about it to the README.